pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go

```diff
@@ -606,6 +609,191 @@ func testOperatorStateTransitions(events monitorapi.Intervals, conditionTypes []
+					return "https://issues.redhat.com/browse/OCPBUGS-62627"
+				}
+			case "kube-storage-version-migrator":
+				if reason == "KubeStorageVersionMigrator_Deploying" {
+					return "https://issues.redhat.com/browse/OCPBUGS-62629"
+				}
+			case "network":
+				if reason == "Deploying" {
+					return "https://issues.redhat.com/browse/OCPBUGS-62630"
+				}
+			case "node-tuning":
+				if reason == "Reconciling" {
+					return "https://issues.redhat.com/browse/OCPBUGS-62632"
+				}
+			case "openshift-controller-manager":
+				if reason == "_DesiredStateNotYetAchieved" {
+					return "https://issues.redhat.com/browse/OCPBUGS-63116"
+				}
+			case "service-ca":
+				if reason == "_ManagedDeploymentsAvailable" {
+					return "https://issues.redhat.com/browse/OCPBUGS-62633"
+				}
+			case "storage":
+				// GCPPDCSIDriverOperatorCR_GCPPDDriverControllerServiceController_Deploying
+				// GCPPDCSIDriverOperatorCR_GCPPDDriverNodeServiceController_Deploying
+				// AWSEBSCSIDriverOperatorCR_AWSEBSDriverNodeServiceController_Deploying
+				// VolumeDataSourceValidatorDeploymentController_Deploying
+				if strings.HasSuffix(reason, "Controller_Deploying") ||
+					reason == "GCPPD_Deploying" {
+					return "https://issues.redhat.com/browse/OCPBUGS-62634"
+				}
+			case "olm":
+				// CatalogdDeploymentCatalogdControllerManager_Deploying
+				// OperatorcontrollerDeploymentOperatorControllerControllerManager_Deploying
+				if strings.HasSuffix(reason, "ControllerManager_Deploying") {
+					return "https://issues.redhat.com/browse/OCPBUGS-62635"
+				}
+			}
+			return ""
+		}
+
+		var excepted, fatal []string
+		for _, operatorEvent := range operatorEvents {
+			if operatorEvent.From.Before(machineConfigProgressing) {
+				continue
+			}
+			condition := monitorapi.GetOperatorConditionStatus(operatorEvent)
+			if condition == nil {
+				continue // ignore non-condition intervals
+			}
+			if condition.Type == "" {
+				fatal = append(fatal, fmt.Sprintf("failed to convert %v into a condition with a type", operatorEvent))
+				continue
+			}
+
+			if condition.Type != configv1.OperatorProgressing || condition.Status == configv1.ConditionFalse {
+				continue
+			}
+
+			// if there was any switch, it was wrong/unexpected at some point
+			failure := fmt.Sprintf("%v", operatorEvent)
+
+			exception := except(operatorName, condition.Reason)
+			if exception == "" {
+				fatal = append(fatal, failure)
+			} else {
+				excepted = append(excepted, fmt.Sprintf("%s (exception: %s)", failure, exception))
+			}
+		}
+
+		output := fmt.Sprintf("%d (out of %d) unexpected clusteroperator state transitions while machine-config is progressing during the upgrade window from %s to %s", len(fatal), len(operatorEvents), start.Format(time.RFC3339), stop.Format(time.RFC3339))
+		if len(fatal) > 0 {
+			output = fmt.Sprintf("%s.  These did not match any known exceptions, so they cause this test-case to fail:\n\n%v\n", output, strings.Join(fatal, "\n"))
```

Comment:
fatal include all operators but we create test cases per operator, thus its possible to have fatal erros not relevant to an operator<<